app/views/projects/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Home Page</h1>
```

Comment:
So, an index page isn't necessarily "the home page" - as I wrote above, the index page for a given resource is usually the big list of all the instances of that resource. 

This page wasn't required by the user stories, so next time I would leave it out unless you are practicing something specific.